[core]/esx_context/index.html

```diff
@@ -390,8 +390,8 @@
       });
 
       window.addEventListener("keydown", (e) => {
-        if (e.key == "Escape" || e.key == "Backspace") {
-          if (e.target.tagName.toLowerCase() == "input") {
+        if (e.key === "Escape" || e.key === "Backspace") {
```

Comment:
may i know why ===  or miss type  ? for are they same???

Reply:
> may i know why === or miss type ? for are they same???

I didn't understand your comment

Reply:
=== is faster since == checks for the type of the variable and then compares while === checks if they are equal